tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/arrays/SealedArrayTest.java

```diff
@@ -282,51 +222,99 @@ public void testNegativeSealedArrays() {
         BAssertUtil.validateError(resultNegative, 0, "variable 'sealedArray1' is not initialized", 19, 5);
     }
 
+    // TODO : uncomment
+    @Test()
+    public void testNegativeAutoFillSealedArray() {
+        Assert.assertEquals(listExprNegative.getErrorCount(), 11);
+        BAssertUtil.validateError(listExprNegative, 0,
+                                  "invalid usage of list constructor: type 'Person[5]' does not have a filler value",
+                                  24,
+                                  19);
```

Comment:
Can we have these in a single line?

Reply:
Fixed https://github.com/ballerina-platform/ballerina-lang/pull/20749/commits/b76a0ffddd265c6a2226339c76801d5aff39ffb9